e2e/scripts/generate-project.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+const { runGeneratorWithResponses } = require("./execute-shell-commands.js");
+const config = require("../config.js");
+const { program, Argument } = require("commander");
+const { mkdirIfNotExists } = require("./utils.js");
+
+const main = async (opts) => {
+  const { args } = opts;
+  const [project] = args;
+  if (opts.args.length !== 1) {
+    console.log(program.helpInformation());
+    process.exit(1);
+  }
+
+  try {
+    // Explicitly create outputDir because generator runs into permissions issue when generating no-ext projects.
+    await mkdirIfNotExists(config.GENERATED_PROJECTS_DIR);
+    const outputDir = `${config.GENERATED_PROJECTS_DIR}/${project}`;
+    // TODO: Update script to setup local verdaccio npm repo to allow running 'npx @salesforce/pwa-kit-create-app' to generate apps
```

Comment:
e.g. right now you're calling the binary directly from within the repo? I think `npx` might be a bit slower and not sure if it's necessary or not. Calling the binary directly might be fine

Reply:
Yup makes sense. That was more of a feedback on previous work considering we were trying to get as close to real-user simulation as possible.